python/lsst/dax/apdb/tests/data_factory.py

```diff
@@ -141,6 +142,9 @@ def makeSourceCatalog(
     """
     nrows = len(objects)
     midpointMjdTai = visit_time.mjd
+    # Note that for now we use naive datetime for time_processed, to have it
+    # consistent with ap_association.
+    time_processed = datetime.datetime.now()
```

Comment:
This seems to be a step in the wrong direction, by propagating buggy behavior upstream from `ap_association`. This would all need to be undone once the bug was fixed.

If you really insist on taking this route, please make it a TODO and Jira issue, or it will never actually get fixed.

Reply:
This will be fixed when we update ap_association to use aware timestamp. We do need a new ticket for that, even with my current changes the situation is not perfect, I'll open a Jira ticket.